poster/index.html

```diff
@@ -1,114 +1,108 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="de">
 
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>poster</title>
+    <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:100|Megrim|Raleway:200,900" rel="stylesheet">
```

Comment:
Your original Google fonts links were separate, but you can combine them like so and this will cut down on the requests your HTML file needs to make 🙌 (original version makes 9 requests, this one makes 6).